vsintegration/src/.gitignore

```diff
@@ -0,0 +1 @@
+TestResults
```

Comment:
when i run unit test from inside visual studio, this is the folder created.
I'll double check